app/javascript/gobierto_dashboards/webapp/containers/contracts/Summary.vue

```diff
@@ -176,12 +228,168 @@ export default {
       labelProcessType: I18n.t('gobierto_dashboards.dashboards.contracts.process_type'),
       labelAmountDistribution: I18n.t('gobierto_dashboards.dashboards.contracts.amount_distribution'),
       labelMainAssignees: I18n.t('gobierto_dashboards.dashboards.contracts.main_assignees'),
+      labelBeesWarm: I18n.t('gobierto_dashboards.dashboards.visualizations.title_beeswarm'),
+      labelTooltipBeesWarm: I18n.t('gobierto_dashboards.dashboards.visualizations.tooltip_beeswarm'),
+      labelMultipleLine: I18n.t('gobierto_dashboards.dashboards.visualizations.title_multiple'),
+      queryLineChart: "?sql=SELECT final_amount_no_taxes, status, initial_amount_no_taxes, start_date FROM contratos WHERE contract_type != 'Patrimonial'",
+      dataBeesWarm: undefined,
+      dataBeesWarmFilter: undefined,
+      dataLineChart: undefined,
+      valuesForLineChart: undefined,
+      valuesForCircleChart: undefined,
+      labelRootKey: I18n.t('gobierto_dashboards.dashboards.contracts.assignees'),
     }
   },
-  created() {
+  watch: {
+    dashboardsData(newValue, oldValue) {
+      if (newValue !== oldValue) {
+        this.updateDataBeesWarm(newValue)
+      }
+    }
+  },
+  async created() {
     this.columns = assigneesColumns;
+    this.dataBeesWarmFilter = JSON.parse(JSON.stringify(this.dashboardsData));
+
+    const { data: { data: dataLineChart } } = await getQueryData(this.queryLineChart)
+    this.transformDataContractsLine(dataLineChart)
   },
   methods: {
+    updateDataBeesWarm(data){
+      const dataBeesWarm = JSON.parse(JSON.stringify(data));
+      this.dataBeesWarmFilter = dataBeesWarm
+    },
+    transformDataContractsLine(data) {
+      data.forEach(d => {
+        d.final_amount_no_taxes = +d.final_amount_no_taxes
+        d.initial_amount_no_taxes = +d.initial_amount_no_taxes
+        //Calculate percentage median between initial amount and final amount to obtain the difference.
+        d.year = new Date(d.start_date).getFullYear()
+
+        if (d.status === "Formalizado" || d.status === "Adjudicado") {
+          d.formalized = 1
+        } else {
+          d.formalized = 0
+        }
+
+        if (d.status === "Desierto") {
+          d.anulled = 1
+        } else {
+          d.anulled = 0
+        }
+
+        if (d.final_amount_no_taxes === 0) {
+          d.formalized = 0
+          d.anulled = 1
+        }
+      })
+
+      //JS convert null years to 1970
+      let NEXT_YEAR = new Date()
+      let GET_NEXT_YEAR = NEXT_YEAR.getFullYear() + 1
+      data = data.filter(({ year }) => year !== 1970 && year !== GET_NEXT_YEAR)
```

Comment:
I don't understand your naming logic. Try to set variables as they properly correspond.

You name a variable `NEXT_YEAR` but you set `new Date()` inside. In any case, this variable could be named as CURRENT_YEAR. But, you're only using this var to calculate the following variable, so it could be removed. Next, you use a transitive verb to define a variable name, GET_NEXT_YEAR, but it's not a function.

Wouldn't be better, clean code the following?

```suggestion 
      const NEXT_YEAR = new Date().getFullYear() + 1
      data = data.filter(({ year }) => year !== 1970 && year !== NEXT_YEAR)
```